### PR TITLE
Update string-not_translatable.xml

### PR DESCRIPTION
**Pull request by def14nt**
Changed some small things with the values, see commits

### CODE REVIEW

File: app/src/main/res/values/string-not_translatable.xml [hunk 1]
```diff
@@ -82,14 +82,11 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     \n}
     \n</style>
     \n
-    \n
```
Comment (gsantner) [inline app/src/main/res/values/string-not_translatable.xml]: the additional empty lines are there to faster see that these are different blocks. keep it.

as pf JS, keep it with 2 indent. I don't like 2space either, but on mobile you hardly see something in one line with spaces in front.

**Comment by def14nt** [comment]
@gsanter comments? What's wrong with it. Just a small annoyance

**Comment by gsantner** [comment]
I dont want it to be like you commited and said that.

**Comment by def14nt** [comment]
Oh shit, actually didn't see that review. Surprised the FastHub team haven't fixed that yet :/

**Comment by def0x099** [comment]
@gsantner 

> I dont want it to be like you commited and said that

What does this mean, by the way? I'm actually still curious whenever I come across this PR. :+1:

**Pull request closed without merging**